src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/PlayerPostDTO.java

```diff
@@ -0,0 +1,14 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto;
+
+public class PlayerPostDTO {
```

Comment:
Idk if we already implemented the change name, ready stuff for the player, but if not there may be a name clash with that DTO.
Also as we are just sending the invitationCode wouldn't something like InvitationCodePostDTO be better?

Reply:
Sorry I don't understand - I just named it this way because it is posting a player to a lobby and it seemed consistent. Is there some other instance of posting a player (e.g. inside the game where this name would be more appropriate?) If so, I can rename.

Reply:
Ah no it's okey, I checked it again, the other one was the PlayerPutDTO, and your reasoning makes sense 👍